src/v2i-hub/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ SET (TMX_APPNAME tmx CACHE INTERNAL "Application name")
 SET (TMX_APPDESCR "V2I Hub" CACHE INTERNAL "Application description")
 LIST(APPEND CMAKE_PREFIX_PATH "/opt/carma/lib")
 LIST(APPEND CMAKE_MODULE_PATH "/opt/carma/cmake")
-
+LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}")
```

Comment:
Are all the cmake modules in the CMAKE_SOURCE_DIR?

Reply:
Line 19 checks for cmake configs installed via the v2i-hub or tmx CMakeLists. `/opt/carma/cmake` checks for apt installed cmake packages

Reply:
I see. Are cmake configs also installed in subdirectories of `${CMAKE_SOURCE_DIR} `folder? Does this `APPEND` search for all cmakes under the subdirectories?